### DISCUSSION TITLE
Regarding Miele-LXIV

### DISCUSSION THREAD

**Discussion by angelaxdd** [bettar/miele-lxiv#58] [General]
I am an user of Miele-LXIV! I am currently using it to view angiograms. I would like know how I could know the orientation of each angiogram (ie RAO/LAO view & cranial/caudal view). Thank you very much!!

**Comment by zdavatz** [comment]
Are you talking about 2D or 3D? There should be marks on the image. So yes it is possible.

**Comment by zdavatz** [comment]
@angelaxdd says: I’m asking about this https://www.wikidoc.org/index.php/Coronary_angiography_left_coronary_artery
There are some sample images here
I would like to know about how do I see the orientations ie RAO, LAO etc

**Comment by zdavatz** [comment]
![Bildschirmfoto 2020-08-17 um 13 52 32](https://user-images.githubusercontent.com/4953/90393429-065f2580-e091-11ea-9d92-5e0231a7e008.png)

<img width="598" alt="Bildschirmfoto 2020-08-17 um 13 53 38" src="https://user-images.githubusercontent.com/4953/90393458-1840c880-e091-11ea-9d0b-9aa41e976679.png">

**Comment by zdavatz** [comment]
images from here: https://www.medicalconnections.co.uk/Sample-Images/

**Comment by zdavatz** [comment]
![Bildschirmfoto 2020-08-17 um 14 06 27](https://user-images.githubusercontent.com/4953/90394418-e892c000-e092-11ea-9cee-6ad560559dad.png)

https://www.medicalconnections.co.uk/download/Images/MultiFrame.zip

**Comment by zdavatz** [comment]
you can select the oritentations here: 
<img width="1440" alt="Bildschirmfoto 2020-08-17 um 14 39 57" src="https://user-images.githubusercontent.com/4953/90397095-9ef8a400-e097-11ea-89ff-75ec4782fa8d.png">

**Comment by zdavatz** [comment]
<img width="1440" alt="Bildschirmfoto 2020-08-17 um 14 43 01" src="https://user-images.githubusercontent.com/4953/90397400-13334780-e098-11ea-8a96-c0576653eab6.png">

**Comment by zdavatz** [comment]
[Angela_Test-Image.DCM.zip](https://github.com/bettar/miele-lxiv/files/5095765/Angela_Test-Image.DCM.zip)

**Comment by zdavatz** [comment]
https://www.microdicom.com/ shows this:

![li](https://user-images.githubusercontent.com/4953/90633903-2d4f6000-e227-11ea-90b7-529835146185.jpeg)
